tox.ini

```diff
@@ -1,4 +0,0 @@
-[flake8]
-max-line-length = 120
-ignore = F401,E402,F403,W503,W504
-exclude = docs/src
```

Comment:
Since it was there in this file I thought let's keep it

Reply:
This file is very outdated I think, we shouldn't trust whatever is in it. Let's please keep setup.cfg as it was

Reply:
Reverted.